source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -118,8 +124,13 @@ def test_autocomplete_call_override(self):
         """"
         test that the _autocomplete param switch the right autocomplete service
         """
+        url = 'https://host_of_bragi/autocomplete'
+        kwargs = {'params': {u'q': u'bob', u'type[]': [u"['stop_area', 'address', 'poi', 'administrative_region']"],
```

Comment:
administrative_region = city in bragi

do we need to check bragi types as we do here https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/autocomplete/kraken.py#L64 ?